ApeWorX/ethpm-types
# fix: issue where objects using HexBytes could not serialize [APE-1226]

### What I did

Having the toughest time converting `TraceFrames` from `evm_trace` into JSON.
Figured it was best to fix at the source so this wouldnt happen anywhere else.

### How I did it

Create a custom dumps method that handles HexBytes

### How to verify it

Create a model with a sub-dict using HexBytes as its keys
Make sure can call `.json()` on that model.

### Checklist
